test/mocks/api/mocks.cc

```diff
@@ -60,8 +60,6 @@ MockOsSysCalls::~MockOsSysCalls() = default;
 
 SysCallIntResult MockOsSysCalls::setsockopt(os_fd_t sockfd, int level, int optname,
                                             const void* optval, socklen_t optlen) {
-  ASSERT(optlen == sizeof(int));
```

Comment:
Oh, I see why this ASSERT was here. On the 2nd to last line of the function, `optval` is cast to `int`. Can you make that line be conditional on the size either being equal to int, or >=?

Reply:
Done